R/create_default_parameters.R

```diff
@@ -13,7 +13,7 @@
 #'   applied for each fleet that is not specified but present in `data` based
 #'   on the types of information present for that fleet.
 #' @param recruitment A list specifying the settings for the recruitment
-#'   module. The default is a Beverton--Holt recruitment relationship with
+#'   module. The default is a Beverton-Holt recruitment relationship with
```

Comment:
@Bai-Li-NOAA I am curious why you made this and other associated changes. An en-dash is appropriate between last names of individuals and between contrasting comparisons/relationships. I cannot remember if roxygen parses these into true en-dashes, i.e., a dash the width of an "n" rather than two dashes that are used to approximate the width of an n and differentiate it from a regular dash or an em-dash.

Reply:
@kellijohnson-NOAA. Thanks for explaining the reason behind the use of `--`. I made the changes initially because when I printed the `estimates` tibble in R, it displayed `Beverton--Holt`, which I assumed was a typo. I'll revisit the changes I made and also review the rendered files to confirm how it’s being parsed.

Reply:
I've reverted the change from Beverton--Holt to Beverton-Holt. Thanks for pointing that out.